docs/EN_US/HPCCClientTools/CT_Mods/ECLCC.xml

```diff
@@ -619,8 +619,10 @@
               <row>
                 <entry>--metacache=x</entry>
 
-                <entry>Specify directory to store distributed meta
-                information</entry>
+                <entry>Specify directory to store distributed meta information
+                from the eclcc indexer. To disable the indexer, set to an
+                empty value. If omitted, the default location is
```

Comment:
I think the text is fine.  I would explictly include the option to disable it since it may not be obvious.  E.g.

    set to an empty value using "--metacache=".  If...